modules/mobile/app/controllers/mobile/v0/claims_and_appeals_controller.rb

```diff
@@ -5,12 +5,24 @@
 require_relative '../../../models/mobile/v0/adapters/claims_overview_errors'
 require_relative '../../../models/mobile/v0/claim_overview'
 require 'sentry_logging'
+require 'prawn'
+require 'fileutils'
+require 'mini_magick'
 
 module Mobile
   module V0
     class ClaimsAndAppealsController < ApplicationController
       include IgnoreNotFound
+      UUID = SecureRandom.uuid
+      TMP_BASE_PATH = Rails.root.join 'tmp', 'uploads', 'cache', UUID
```

Comment:
Although the random UUID is added this is still constant for the duration of the the app run on the instance of the server. Could that part of the directory path creation move into the `upload_documents` method? You can set it with a class instance variable `@variable_name` that way the `after_action` hook will have it in scope.

Reply:
Gotcha, thanks